lambdas/session/src/main/java/uk/gov/di/ipv/cri/address/api/handler/SessionHandler.java

```diff
@@ -4,31 +4,42 @@
 import com.amazonaws.services.lambda.runtime.RequestHandler;
 import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
 import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
-import com.google.gson.Gson;
-import org.apache.http.HttpStatus;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
 import software.amazon.lambda.powertools.logging.CorrelationIdPathConstants;
 import software.amazon.lambda.powertools.logging.Logging;
 import software.amazon.lambda.powertools.metrics.Metrics;
+import uk.gov.di.ipv.cri.address.library.domain.SessionRequest;
+import uk.gov.di.ipv.cri.address.library.error.ErrorResponse;
+import uk.gov.di.ipv.cri.address.library.exceptions.ServerException;
+import uk.gov.di.ipv.cri.address.library.exceptions.ValidationException;
+import uk.gov.di.ipv.cri.address.library.helpers.ApiGatewayResponseGenerator;
+import uk.gov.di.ipv.cri.address.library.helpers.EventProbe;
 import uk.gov.di.ipv.cri.address.library.service.AddressSessionService;
 
 import java.util.Map;
+import java.util.UUID;
+
+import static org.apache.logging.log4j.Level.ERROR;
+import static org.apache.logging.log4j.Level.INFO;
+import static software.amazon.awssdk.http.HttpStatusCode.BAD_REQUEST;
```

Comment:
Elsewhere, we're using org.apache.http.HttpStatus for status codes. We should pick one and standardise

Reply:
I think I prefer the `aws` so maybe we can switch to Phil use here

Reply:
I'm ok with switching to `org.apache.http.HttpStatus` so will do this